.github/workflows/build_linux_packages.yml

```diff
@@ -53,39 +53,50 @@ jobs:
       - name: Checkout Repository
         uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
 
+      - name: Install python deps
+        run: |
+          pip install -r requirements.txt
+
       # TODO: We shouldn't be using a cache on actual release branches, but it
       # really helps for iteration time.
-      - name: Runner Health Settings
+      - name: Setup ccache
         run: |
           ./build_tools/setup_ccache.py \
             --config-preset "github-oss-presubmit" \
             --dir "$(dirname $CCACHE_CONFIGPATH)" \
             --local-path "$CACHE_DIR/ccache"
+          ccache -z
+
+      - name: Runner Health Status
+        run: |
           df -h
           echo "ccache version: $(ccache --version)"
           echo "ccache config:"
           echo "---"
           cat "$CCACHE_CONFIGPATH"
           echo "---"
-          ccache -z
```

Comment:
I think this was explicitly desired here and the below is just to check that stat counters have been cleared in order to report statistics for the particular workflow at the end.

Reply:
Zeroing the cache should be done by `build_tools/setup_ccache.py`.

Sequencing is a bit tricky. How about
* Health checks show that ccache is installed and report its ccache
* setup_ccache.py writes the config, zeroes stats, and logs whatever it does

I would also prefer for the long versions of flags to be used, like `--zero-stats` instead of `-z`. See the docs at https://ccache.dev/manual/latest.html#_command_line_options

Reply:
ok :+1: I would suggest that we merge it how it is right now and then i have another PR with all the changes in the python scripts: `runner_health_status.py` and `build_tools/setup_ccache.py`